src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -234,13 +245,16 @@ var styles = StyleSheet.create({
   },
   generalButtonStyle: {
     alignItems: 'center',
-    backgroundColor: colors.gray60,
     borderRadius: 50,
     justifyContent: 'center',
     position: 'absolute',
     zIndex: 1
   },
+  opacity: {
+    opacity: 0.6
+  },
   screenRecording: {
+    backgroundColor: colors.borderRgba,
```

Comment:
please use `surface` to match the white from figma

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/577/commits/0e3b8d076de6807d14cb8ba6734ed61148f5e196